Codegen/src/Nodes/FunctionDeclarationCG.cs

```diff
@@ -132,7 +132,8 @@ internal class FunctionDeclarationCG : Compiler.Nodes.FunctionDeclaration, Gener
                 foreach (var proc in pgm.Procedures.Values) {
                     proc.IsNotByExternalPointer = true;
                     toAddRange.Add(new GeneratedNode2(" ", true));
-                    toAddRange.Add(new GeneratedNode2("*" + pgm.Name + "::" + proc.Name, true));
+                    toAddRange.Add(new GeneratedNode2("*To call program " + proc.Hash + proc.Name + " in module " + proc.ProcStyleCall.FunctionDeclaration.QualifiedName.Tail, false));
+                    toAddRange.Add(new GeneratedNode2("*Which is generated code for " + proc.ProcStyleCall.FunctionDeclaration.QualifiedName, false));
```

Comment:
My code example in the issue was missing line ```*Declared in source file XXXX```.
Can you add it please ? (like you did in skeleton file).

I'll edit the issue to add this line.